modules/operator-unmanaged-hpa.adoc

```diff
@@ -6,21 +6,105 @@ Horizontal Pod Autoscalers (HPAs) are now included with the `Clair`, `Quay`, and
 
 As HPA is configured by default to be managed, the number of `Clair`, `Quay`, and `Mirror` pods is set to two. This facilitates the avoidance of downtime when updating or reconfiguring {productname} through the Operator or during rescheduling events.
 
+[NOTE]
+====
+There is a known issue when disabling the `HorizontalPodAutoscaler` component and attempting to edit the HPA resource itself and increase the value of the `minReplicas` field. When attempting this setup, `Quay` application pods are scaled out by the unmanaged HPA and, after 60 seconds, the replica count is reconciled by the {productname} Operator. As a result, HPA pods are continuously created and then removed by the Operator. 
+
+To resolve this issue, you should upgrade your {productname} deployment to at least version 3.12.5 or 3.13.1 and then use the following example to avoid the issue.
+====
+
 [id="operator-disabling-hpa"]
 == Disabling the Horizontal Pod Autoscaler
 
-To disable autoscaling or create your own `HorizontalPodAutoscaler`, specify the component as `unmanaged` in the `QuayRegistry` instance. For example:
+To disable autoscaling or create your own `HorizontalPodAutoscaler` component, specify the component as `unmanaged` in the `QuayRegistry` custom resource definition. To avoid the known issue noted above, you must modify the `QuayRegistry` CRD object and set the replicas equal to `null` for the `quay` component. 
+
+.Procedure
 
+* Edit the `QuayRegistry` CRD to include the following `replicas: null` for the `quay` component:
++
+[source,terminal]
+----
+$ oc edit quayregistry <quay_registry_name> -n <quay_namespace>
+----
++
 [source,yaml]
 ----
 apiVersion: quay.redhat.com/v1
 kind: QuayRegistry
 metadata:
-  name: example-registry
-  namespace: quay-enterprise
+name: quay-registry
+namespace: quay-enterprise
 spec:
-  components:
-    - kind: horizontalpodautoscaler
-      managed: false
+components:
+  - kind: horizontalpodautoscaler
+    managed: false
+  - kind: quay
+    managed: true
+    overrides: 
+      replicas: null <1>
 # ...
+----
+<1> After setting `replicas: null` in your `QuayRegistry` CRD, a new replica set might be generated because the deployment manifest of the `Quay` app is changed with `replicas: 1`. 
+
+.Verification
+
+. Create a customized `HorizontalPodAutoscalers` CRD and increase the `minReplicas` amount to a higher value, for exampe, `6`:
```

Comment:
for exampe, `6` --> suggest to: for exampe, `3`, as below examples all shows 3 replicas

Reply:
Changed to 3